src/views/states/list/components/utils.ts

```diff
@@ -31,6 +31,11 @@ export const interfaceFilters: Record<
       status === LLDP_ENABLED ? Boolean(obj?.lldp?.enabled) : !obj?.lldp?.enabled,
     );
   },
+  [FILTER_TYPES.MAC_ADDRESS]: (selectedInput, obj) => {
+    const searchMACAddress = selectedInput?.[0];
```

Comment:
Is it necessary to create this const? Just asking...

Reply:
we can remove that

Reply:
fixed